wrapper/src/test/build.gradle.kts

```diff
@@ -81,5 +81,10 @@ tasks.withType<Test> {
 }
 
 tasks.register<Test>("in-container") {
+    filter.excludeTestsMatching("software.*") // exclude unit tests
```

Comment:
this task runs the unit tests as well?

Reply:
With the exact code in this PR, this line isn't needed. However, if I use a filter like `*ReadWriteSplittingTests` I get a compilation error in one of the unit tests. This was unexpected but I think the * in the filter somehow triggers compilation of the unit tests. For reference, the error is `java.lang.ClassNotFoundException: software.amazon.awssdk.services.secretsmanager.model.GetSecretValueRequest`